modules/monitoring-configuring-notifications-for-default-platform-alerts.adoc

```diff
@@ -0,0 +1,134 @@
+
+. Edit the Alertmanager configuration:
++
+[source,yaml]
+----
+global:
+  resolve_timeout: 5m
+route:
+  group_wait: 30s #<1>
+  group_interval: 5m #<2>
+  repeat_interval: 12h #<3>
+  receiver: default
+  routes:
+  - matchers:
+    - "alertname=Watchdog"
+    repeat_interval: 2m
+    receiver: watchdog
+  - matchers:
+    - "service=<your_service>" #<4>
+    routes:
+    - matchers:
+      - <your_matching_rules> #<5>
+      receiver: <receiver> #<6>
+receivers:
+- name: default
+- name: watchdog
+- name: <receiver>
+  <receiver_configuration> #<7>
+----
+<1> Specify how long Alertmanager waits while collecting initial alerts for a group of alerts before sending a notification.
+<2> Specify how much time must elapse before Alertmanager sends a notification about new alerts added to a group of alerts for which an initial notification was already sent.
+<3> Specify the minimum amount of time that must pass before an alert notification is repeated.
+If you want a notification to repeat at each group interval, set the `repeat_interval` value to less than the `group_interval` value.
+However, the repeated notification can still be delayed, for example, when certain Alertmanager pods are restarted or rescheduled.
+<4> Specify the service that fires the alerts.
+<5> Specify labels to match your alerts.
+<6> Specify the receiver to use for the alerts.
+<7> Specify the receiver configuration.
++
+[IMPORTANT]
+====
+* Use the `matchers` key name to indicate the matchers that an alert has to fulfill to match the node.
+Do not use the `match` or `match_re` key names, which are both deprecated and planned for removal in a future release.
+
+* If you define inhibition rules, use the following key names:
++
+--
+** `target_matchers`: to indicate the target matchers
+** `source_matchers`: to indicate the source matchers
+--
++
+Do not use the `target_match`, `target_match_re`, `source_match`, or `source_match_re` key names, which are deprecated and planned for removal in a future release.
+====
++
+The following Alertmanager configuration example configures PagerDuty as an alert receiver:
++
+[source,yaml]
+----
+global:
+  resolve_timeout: 5m
+route:
+  group_wait: 30s
+  group_interval: 5m
+  repeat_interval: 12h
+  receiver: default
+  routes:
+  - matchers:
+    - "alertname=Watchdog"
+    repeat_interval: 2m
+    receiver: watchdog
+  - matchers:
+    - "service=example-app"
+    routes:
+    - matchers:
+      - "severity=critical"
+      receiver: team-frontend-page
+receivers:
+- name: default
+- name: watchdog
+- name: team-frontend-page
+  pagerduty_configs:
+  - service_key: "_your-key_"
+----
++
+With this configuration, alerts of `critical` severity that are fired by the `example-app` service are sent by using the `team-frontend-page` receiver. Typically, these types of alerts would be paged to an individual or a critical response team.
```

Comment:
We call this PagerDuty above, but here say that the receiver is the `team-frontend-page` receiver. You could clarify the difference (that is implied) between the "notification system" and "alert receiver" in this procedure. (Or clarify that one is the name users see and one is the name of the config file or Kind or whatever it is.)

There seems to be some nuance that has been lost starting with https://79966--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/managing-alerts.html#sending-notifications-to-external-systems_managing-alerts when it comes to defining the "notification system" and "alert receiver." The language switches without any explanation and continues here. A potential area for later improvement/clarification.

Reply:
Yes, this will all be addressed later and I am aware of this, see this comment for explanation, thanks!
https://github.com/openshift/openshift-docs/pull/79966#discussion_r1721365506